plio/tests.py

```diff
@@ -993,6 +993,136 @@ def test_question_metrics_answers_provided(self):
         self.assertEqual(response.data["accuracy"], 58.33)
         self.assertEqual(response.data["has_survey_question"], False)
 
+    def test_question_mode_returned_if_no_sessions(self):
```

Comment:
the test is already under `QuestionTestCase` - so, you don't need to repeat the word `question` here unless there is a specific reason - the existing tests where `question` word has been used in the function name has been done so to distinguish the question-based metrics from the non-question based metrics (like retention, average watch-time, etc.)